cmd/peer-rest-client.go

```diff
@@ -676,6 +676,32 @@ func (client *peerRESTClient) BackgroundHealStatus() (madmin.BgHealState, error)
 	return state, err
 }
 
+func (client *peerRESTClient) getLocalDiskIDsWith() chan []string {
```

Comment:
perhaps you want to name it as getLocalDiskIDsCh() ?

Reply:
oops yes